cvmfs/cvmfs.cc

```diff
@@ -487,6 +506,7 @@ static void cvmfs_getattr(fuse_req_t req, fuse_ino_t ino,
 
   fuse_remounter_->fence()->Enter();
   ino = mount_point_->catalog_mgr()->MangleInode(ino);
+  TraceInode(Tracer::kEventGetAttr, ino, "getattr()");
```

Comment:
I think this should move down, until after the checks and just before leaving the fence.